client/tx/tx_test.go

```diff
@@ -418,7 +418,7 @@ func TestPreprocessHook(t *testing.T) {
 	txb, err := txfDirect.BuildUnsignedTx(msg1, msg2)
 	requireT.NoError(err)
 
-	err = tx.Sign(nil, txfDirect, from, txb, false) //nolint:staticcheck
+	err = tx.Sign(nil, txfDirect, from, txb, false) //nolint:staticcheck // here we test the old sign call, against the new one, so we need to use the old one.
```

Comment:
ditto

Reply:
I think tx.Sign is deprecated :grimacing:

Reply:
no that is incorrect -- it was angry about a nil context ser